src/app.js

```diff
@@ -598,5 +599,12 @@ app.useAsync(async (err, req, res, next) => {
   return next(err);
 });
 
+/**
+ * Update the resources we know about. If the underlying data (a JSON file on
+ * S3) hasn't changed then this only involves a HEAD request so we run it on an
+ * hourly interval.
+ */
+updateResourceVersions();
+setInterval(updateResourceVersions, 60*60*1000)
```

Comment:
No problem. I've changed the code (and rebased to get the changes from #746) so that we can now set:
* RESOURCE_INDEX="false" to skip resource collection parsing
* RESOURCE_INDEX="path/to/local/json" to load a local index